src/components/chips/HookCreatedChip.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+
+import Avatar from '@material-ui/core/Avatar';
+import Chip from '@material-ui/core/Chip';
+import Tooltip from '@material-ui/core/Tooltip';
+import Icon from '@material-ui/core/Icon';
+import { useTaskStatusColor } from '../../utils/colors';
+import { taskStatusIconName } from '../../utils/status';
+import { roundAndPresentDuration } from '../../utils/time';
+import { createFragmentContainer } from 'react-relay';
+import { graphql } from 'babel-plugin-relay/macro';
+import { useTheme } from '@material-ui/core';
+import { HookCreatedChip_hook } from './__generated__/HookCreatedChip_hook.graphql';
+
+interface Props {
+  hook: HookCreatedChip_hook;
+  className?: string;
+}
+
+let HookCreatedChip = (props: Props) => {
```

Comment:
I wish we could further generalize this component, but it heavily depends on the GraphQL data, so it's either a `Task` or `Hook` or something else.

@fkorotkov any hints how this can be accomplished?